app/views/design-system/rsd-design-system/design-patterns/footer/index.html

```diff
@@ -0,0 +1,174 @@
+          &lt;/li&gt;
+        &lt;/ul&gt;
+      &lt;/div&gt;
+    &lt;/div&gt;
+    &lt;div class=&quot;govuk-footer__meta&quot;&gt;
+      &lt;div class=&quot;govuk-footer__meta-item govuk-footer__meta-item--grow&quot;&gt;
+        &lt;h2 class=&quot;govuk-visually-hidden&quot;&gt;Support links&lt;/h2&gt;
+        &lt;ul class=&quot;govuk-footer__inline-list&quot;&gt;
+          &lt;li class=&quot;govuk-footer__inline-list-item&quot;&gt;
+            &lt;a class=&quot;govuk-footer__link&quot; href=&quot;#&quot;&gt;
+              Accessibility statement
+            &lt;/a&gt;
+          &lt;/li&gt;
+          &lt;li class=&quot;govuk-footer__inline-list-item&quot;&gt;
+            &lt;a class=&quot;govuk-footer__link&quot; href=&quot;#&quot;&gt;
+              Cookies
+            &lt;/a&gt;
+          &lt;/li&gt;
+          &lt;li class=&quot;govuk-footer__inline-list-item&quot;&gt;
+            &lt;a class=&quot;govuk-footer__link&quot; href=&quot;#&quot;&gt;
+              Privacy notice
+            &lt;/a&gt;
+          &lt;/li&gt;
+        &lt;/ul&gt;
+        &lt;svg aria-hidden=&quot;true&quot; focusable=&quot;false&quot; class=&quot;govuk-footer__licence-logo&quot; xmlns=&quot;http://www.w3.org/2000/svg&quot; viewBox=&quot;0 0 483.2 195.7&quot; height=&quot;17&quot; width=&quot;41&quot;&gt;
+          &lt;path fill=&quot;currentColor&quot; d=&quot;M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145&quot;&gt;&lt;/path&gt;
+        &lt;/svg&gt;
+        &lt;span class=&quot;govuk-footer__licence-description&quot;&gt;
+          All content is available under the
+          &lt;a class=&quot;govuk-footer__link&quot; href=&quot;https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/&quot; rel=&quot;license&quot;&gt;Open
+            Government Licence v3.0&lt;/a&gt;, except where otherwise stated
+        &lt;/span&gt;
+      &lt;/div&gt;
+      &lt;div class=&quot;govuk-footer__meta-item&quot;&gt;
+        &lt;a class=&quot;govuk-footer__link govuk-footer__copyright-logo&quot; href=&quot;https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/&quot;&gt;
+          &copy; Crown copyright
+        &lt;/a&gt;
+      &lt;/div&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+&lt;/footer&gt;
+</code>' %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {{ sample.code('/design-system/rsd-design-system/design-patterns/footer/example.html', 'Footer', 'footer',
```

Comment:
This is loading an iframe which in turn is pulling in the example.html which contains full <html><body> structure which isn't needed here, this is resulting in the example view in the tab applying the width grids etc, within existing grids so looks like it's mobile view. 

Use the full html structure example for the "Open example in a new tab" view, but it's not needed for the preview. I expect this is repeated in all the examples, so revisit them.

Reply:
That makes sense why the components look like that. However this is going to also affect the NCS section since they also use iframes to render their previews